docs/source/deepinv.optim.rst

```diff
@@ -125,13 +125,13 @@ This class comes with methods, such as :math:`\operatorname{prox}_{\distancename
    :template: myclass_template.rst
    :nosignatures:
 
-   deepinv.optim.DataFidelity
-   deepinv.optim.L1
-   deepinv.optim.L2
-   deepinv.optim.IndicatorL2
-   deepinv.optim.PoissonLikelihood
-   deepinv.optim.LogPoissonLikelihood
-   deepinv.optim.AmplitudeLoss
+   deepinv.optim.data_fidelity.DataFidelity
```

Comment:
can we keep the same names for backwards compatibility?

Reply:
Yes!

Reply:
they will continue working without the data_fidelity path right?